src/Services/LDAPService.php

```diff
@@ -488,26 +494,29 @@ public function getLDAPGroupMembers($dn)
      * Constraints:
      * - GUID of the member must have already been set, for integrity reasons we don't allow it to change here.
      *
-     * @param Member
+     * @param Member $member
      * @param array|null $data If passed, this is pre-existing AD attribute data to update the Member with.
      *            If not given, the data will be looked up by the user's GUID.
+     * @param bool $updateGroups controls whether to run the resource-intensive group update function as well. This is
+     *                          skipped during login to reduce load.
```

Comment:
I don't think it's a valid speed optimisation technique to remove a feature :-) This feature is needed - otherwise the users will have to wait until the sync task runs before they can log in, which is not acceptable.

Example scenario:
- user is added to LDAP directly by the system administrator
- sync task has not ran yet (say, because it's hourly, or never runs because the site relies on login-sync)
- user tries to log in
- authorisation fails - because group sync hasn't ran, and the user has no groups.

Reply:
Thanks @mateusz. I had to add the switch locally as it was too slow during local testing. 
I've re-enabled it on LDAPAuthenticator::authenticate() on line 164 👍